src/libraries/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -164,17 +164,17 @@
   <data name="net_io_eof" xml:space="preserve">
     <value> Received an unexpected EOF or 0 bytes from the transport stream.</value>
   </data>
-  <data name="net_io_async_result" xml:space="preserve">
-    <value>The parameter: {0} is not valid. Use the object returned from corresponding Begin async call.</value>
-  </data>
   <data name="net_ssl_io_frame" xml:space="preserve">
     <value>The handshake failed due to an unexpected packet format.</value>
   </data>
   <data name="net_ssl_io_renego" xml:space="preserve">
     <value>The remote party requested renegotiation when AllowRenegotiation was set to false.</value>
   </data>
   <data name="net_ssl_io_cert_validation" xml:space="preserve">
-    <value>The remote certificate is invalid according to the validation procedure.</value>
+    <value>The remote certificate is invalid according to the validation procedure: {0}</value>
+  </data>
+  <data name="net_ssl_io_cert_custom_validation" xml:space="preserve">
+    <value>The remote certificate was rejected by RemoteCertificateValidationCallback.</value>
```

Comment:
Nit re wording: The RemoteCertificateValidationCallback is provided by the user, right? Can we change this to something like "The remote certificate was rejected by the provided RemoteCertificateValidationCallback" or similar, so it's clearer what's going on here?

Reply:
yes, this is user provided. Updated as suggested.